Octokit.Tests/Exceptions/AbuseExceptionTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Octokit.Internal;
+using Xunit;
+
+namespace Octokit.Tests.Exceptions
+{
+    public class AbuseExceptionTests
```

Comment:
@ryangribble makes sense. I'll reformat. Sorry, should've paid more attention to the existing conventions. They tend to be my preference anyway; I think I was likely being hasty.